project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=0.13.16-M1
```

Comment:
sbt 0.13.16 final has been published. Could you amend your PR to use that version, please?